### PR TITLE
Use // for font's url

### PR DESCRIPTION
**Pull request by alicoding**

### CODE REVIEW

**Comment by alicoding** [comment]
@cadecairos R?

File: templates/less/styles.less [hunk 1]
```diff
@@ -3,7 +3,7 @@
 @import "../../node_modules/font-awesome/less/font-awesome";
 @fa-font-path: "/assets/font-awesome/fonts";
 // Open Sans font
-@import url(http://fonts.googleapis.com/css?family=Open+Sans:400,300,400italic,300italic,700);
+@import url(//fonts.googleapis.com/css?family=Open+Sans:400,300,400italic,300italic,700);
```
Comment (jbuck) [inline templates/less/styles.less]: You should force this to be https instead: http://www.paulirish.com/2010/the-protocol-relative-url/

Comment (alicoding) [inline templates/less/styles.less]: Good point!

**Comment by alicoding** [comment]
@jbuck r?

**Pull request merged**